iothub_client/samples/pnp/common/pnp_protocol.c

```diff
@@ -135,7 +135,7 @@ IOTHUB_MESSAGE_HANDLE PnPHelper_CreateTelemetryMessageHandle(const char* compone
 // VisitComponentProperties visits each sub element of the the given objectName in the desired JSON.  Each of these sub elements corresponds to
 // a property of this component, which we'll invoke the application's pnpPropertyCallback to inform.
 // 
-static void VisitComponentProperties(const char* objectName, JSON_Value* value, int version, PnPHelperPropertyCallbackFunction pnpPropertyCallback, void* userContextCallback)
+static void VisitComponentProperties(const char* objectName, JSON_Value* value, int version, PnPPropertyCallbackFunction pnpPropertyCallback, void* userContextCallback)
```

Comment:
nit: should this be `PnP_PropertyCallbackFunction`?

Reply:
Fixing.